pandas/lib.py

```diff
@@ -2,6 +2,7 @@
 
 import warnings
 warnings.warn("The pandas.lib module is deprecated and will be "
```

Comment:
Would it be possible to specially catch `infer_dtype` here and give a more specific message since it has moved and is public?  Noticed I'm using it some real code, I think based on your SO answer here - http://stackoverflow.com/a/20670901/3657742.